html/pfappserver/root/static.alt/src/views/Nodes/_components/NodesSearch.vue

```diff
@@ -119,6 +225,174 @@ export default {
           label: this.$i18n.t('Owner'),
           sortable: true,
           visible: true
+        },
+        {
+          key: 'last_ip',
```

Comment:
I think we need to extend our API to support `last_ip`.

Reply:
The name would change to ip4log.ip